app/pages/index.tsx

```diff
@@ -74,49 +62,31 @@ const Home = ({
     preloadedQuery
   );
 
-  const intakeCalloutChildren = useMemo(() => {
-    if (!openIntake)
-      return (
-        <>
-          <BoldText>Applications are not currently being accepted.</BoldText>
-          <br />
-          Please check the{' '}
-          <Link href="https://www.gov.bc.ca/connectingcommunitiesbc">
-            program webpage
-          </Link>{' '}
-          for updates.
-        </>
-      );
-
-    return (
-      <BoldText>
-        Applications are accepted until{' '}
-        {DateTime.fromISO(openIntake.closeTimestamp).toLocaleString(
-          DateTime.DATETIME_FULL
-        )}
-        .
-        <br />
-        Review of applications will not begin until this date. Draft and
-        submitted applications can be edited until then.
-      </BoldText>
-    );
-  }, [openIntake]);
+  const openIntakeBanner = useFeature('open_intake_alert').value || {};
+  const closedIntakeBanner = useFeature('closed_intake_alert').value || {}; 
 
   return (
     <Layout session={session} title="Connecting Communities BC">
       <div>
         <h1>Welcome</h1>
-        {!openIntake && (
-          <IntakeAlert openTimestamp={nextIntake?.openTimestamp} />
-        )}
```

Comment:
This is the only part of this file that should be updated, the callout below should remain, and the dynamic alerts should be displayed here

Reply:
done